Source/Upstream: Yes, in v238-stable branch

From f619f8a2ff409d20821a54c7faae97cfda30c6c9 Mon Sep 17 00:00:00 2001
From: Yu Watanabe <watanabe.yu+github@gmail.com>
Date: Mon, 12 Mar 2018 17:47:16 +0900
Subject: [PATCH 38/51] busctl: drop redundant pager_open()

(cherry picked from commit 548866015467d82c0688730807bd1b704be21c38)
---
 src/busctl/busctl.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/busctl/busctl.c b/src/busctl/busctl.c
index b4ab0d12d..ae132dffb 100644
--- a/src/busctl/busctl.c
+++ b/src/busctl/busctl.c
@@ -286,8 +286,6 @@ static void print_subtree(const char *prefix, const char *path, char **l) {
 
 static void print_tree(const char *prefix, char **l) {
 
-        pager_open(arg_no_pager, false);
-
         prefix = strempty(prefix);
 
         if (arg_list) {
-- 
2.16.3

